packages/perseus/src/widgets/iframe/iframe.tsx

```diff
@@ -90,10 +96,9 @@ class Iframe extends React.Component<Props> {
         return Changeable.change.apply(this, args);
     };
 
-    simpleValidate: (arg1: any) => any = (rubric) => {
-        // @ts-expect-error - TS2339 - Property 'validate' does not exist on type 'typeof Iframe'.
-        return Iframe.validate(this.getUserInput(), rubric);
-    };
+    simpleValidate(): PerseusScore {
```

Comment:
Comment: I don't _think_ this has to be an arrow function based on how other `simpleValidate` methods are done, but I could be wrong.